app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -32,10 +32,17 @@ const filterByCapabilities = (filterArray, modelToFilter) => API.get(`/api/${mod
         valueArray.push(resource);
       }
     });
-    if (valueArray.length === 0 && modelToFilter !== 'storage_resources') {
-      return [{label: `no ${modelToFilter} with selected capabilities`, value: -1}]
+
+    const options = valueArray.map(({ name, id }) => ({ label: name, value: id }))
+
+    if (options.length === 0 ) {
+      options.unshift({label: `no ${modelToFilter} with selected capabilities`, value: '-1'});
```

Comment:
Minor, but this needs i18n.

Reply:
Does this require something on my part?

Reply:
Rewrite as 
```
sprintf(__('no %s with selected capabilities.'), modelToFilter),
```
So that the string is externalized and we can get it translated